File: ws/acuña/u1/ws05ProgrammingLanguages/tablaMultiplicar.java
public class MultiplicationTable {

    public static void generateMultiplicationTable() {
        for (int i = 1; i <= 10; i++) {
            System.out.println("Tabla de multiplicar del " + i + ":");
            for (int j = 1; j <= 10; j++) {
                System.out.println(i + " x " + j + " = " + (i * j));
            }
            System.out.println(); // Línea en blanco para separar cada tabla
        }
    }

    public static void main(String[] args) {
        generateMultiplicationTable();
    }
}

File: ws/acuña/u1/ws05ProgrammingLanguages/tablaMultiplicar.js
function generateMultiplicationTable() {
    for (let i = 1; i <= 10; i++) {
        console.log(Tabla de multiplicar del ${i}:);
        for (let j = 1; j <= 10; j++) {
            console.log(${i} x ${j} = ${i * j});
        }
        console.log(''); // Línea en blanco para separar cada tabla
    }
}

generateMultiplicationTable();